### DISCUSSION TITLE
Do I need to Install foundry on gitpod

### DISCUSSION THREAD

**Discussion by workrate99** [Cyfrin/foundry-full-course-cu#3069] [Q&A]
I'm using gitpod for this course due to some issues on my laptop, do I still need to Install foundry on the gitpod terminal, cause nothing was said about it in the videos. Thanks for answering

**Comment by EngrPips** [comment]
I don't know much about gitpod, but I think you should try installing it and seeing how it works.

**Comment by workrate99** [comment]
I've tried it and it worked well thanks boss

**Reply by EngrPips** [reply]
Awesome, Sometimes we just need to try it and see.
